Modules/_bisectmodule.c

```diff
@@ -53,9 +53,14 @@ bisect_right(PyObject *self, PyObject *args, PyObject *kw)
     Py_ssize_t index;
     static char *keywords[] = {"a", "x", "lo", "hi", NULL};
 
-    if (!PyArg_ParseTupleAndKeywords(args, kw, "OO|nn:bisect_right",
-        keywords, &list, &item, &lo, &hi))
-        return NULL;
+    if (kw==NULL && PyTuple_GET_SIZE(args)==2) {
```

Comment:
Sorry for be pedantic, but could you please add spaces around `==` for conforming PEP 7? Without spaces the code looks sloppy to me.

Reply:
IIRC, when multiple terms are present, PEP 7 (and PEP 8) allow for tightening spaces on the inner terms while "adding" spaces around the outermost (lowest-priority) operators.

Reply:
I think this is allowed only for arithmetic operators like `*` in `+`.

> Always put spaces around assignment, Boolean and comparison operators. In expressions using a lot of operators, add spaces around the outermost (lowest-priority) operators.